thirdparty/build-thirdparty.sh

```diff
@@ -432,9 +432,8 @@ build_boost() {
     check_if_source_exist $BOOST_SOURCE
     cd $TP_SOURCE_DIR/$BOOST_SOURCE
 
-    echo "using gcc : starrocks : ${CXX} ; " > tools/build/src/user-config.jam
```

Comment:
why delete this line?

Reply:
CC CXX environment variables has been set.